ARCHITECTURE.md

```
@@ -0,0 +1,332 @@
+# Rebuilt2026 Architecture
+*Guerin Robotics — FRC 2025/2026 Competition Robot*
+*Documented post-Worlds. Treat as proven-but-imperfect reference material.*
+
+---
+
+## Project Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)